client/src/components/Dashboard/Dashboard.styled.js

```diff
@@ -0,0 +1,132 @@
+import styled from 'styled-components';
+
+import { $gray, $white, $greyLighter } from '../../assets/styledVars';
+
+export const Container = styled.section`
+	height: 80vh;
+	padding: 0 2.5rem;
+	width: 100%;
+
+	@media (min-width: 1000px) {
+		width: 80%;
+		margin: 0 auto;
+	}
+`;
+
+// TODO: Make Breadcrumbs shared component
```

Comment:
It looks like you did it — this is no longer needed, right?

Reply:
Yeah, forgot to delete it afterward. Will do.